app/assets/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/orthogonal_bucket_picker.js

```diff
@@ -283,6 +116,8 @@ function addNecessaryBucketsToPriorityQueueOrthogonal(
     const additionalPriorityWeight = isFallback ? 1000 : 0;
 
     // Build up priority queue
+    // todo
```

Comment:
Do you want to do this as part of this PR?

Reply:
I think this "todo" is obsolete. At least, I cannot see anything missing here. Do you?